it/mail/chrome/messenger/importMsgs.properties

```diff
@@ -280,78 +280,78 @@
 2135=Note
 
 # Description: Strings for the import dialog
-ImportMailDialogTitle=Import Mail
-ImportAddressBooksDialogTitle=Import Address Books
-ImportSettingsDialogTitle=Import Settings
-ImportMailListLabel=Select application to import mail from:
-ImportAddressBooksListLabel=Select address book format:
-ImportSettingsListLabel=Select application to import settings from:
+ImportMailDialogTitle=Importa e-mail
+ImportAddressBooksDialogTitle=Importa rubriche
+ImportSettingsDialogTitle=Importa impostazioni
+ImportMailListLabel=Seleziona applicazione da cui importare le e-mail:
+ImportAddressBooksListLabel=Seleziona formato rubrica:
+ImportSettingsListLabel=Seleziona applicazione da cui importare le impostazioni:
 
 #Error strings
-ImportAlreadyInProgress=È in corso una operazione di importazione. Riprovare non appena l'importazione corrente è terminata.
+ImportAlreadyInProgress=È in corso una operazione di importazione. Riprova non appena l'importazione corrente è terminata.
 
 #Error strings for settings import
 ImportSettingsBadModule=Impossibile caricare il modulo delle impostazioni
-ImportSettingsNotFound=Impossibile trovare le impostazioni. Assicurarsi che il programma sia installato su questo computer.
-ImportSettingsFailed=Si è verificato un errore nell'importazione delle impostazioni. Parte, o tutte le impostazioni possono non essere state importate.
+ImportSettingsNotFound=Impossibile trovare le impostazioni. Controlla per assicurarti che il programma sia installato su questo computer.
```

Comment:
CORRECTION NEEDED. 

I would shorten the second sentence by replacing "Controlla per assicurarti" with "Assicurati".